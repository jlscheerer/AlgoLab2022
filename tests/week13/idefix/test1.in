20
10 10 28370 7
245 -302
-369 422
-41 -124
201 190
292 -218
302 -117
171 489
-277 -509
-218 361
406 98
-399 407
309 -161
-67 -102
248 -260
194 472
218 -277
-229 371
493 122
-239 396
-66 -125
9 10 1444 6
484 34
-277 24
298 -104
-299 -427
-492 238
98 1
374 -104
262 -94
193 75
484 47
302 -94
192 73
91 -14
277 -92
386 -95
297 -89
375 -116
88 0
373 -87
10 9 11945 4
282 -323
-246 255
282 -3
-332 323
-415 -277
389 205
271 389
220 420
317 -235
-349 338
-266 237
398 197
219 404
-249 276
330 -387
374 193
-407 -292
236 406
-423 -290
10 9 12260 5
451 -133
123 -40
-492 31
266 461
222 -290
202 249
145 -448
341 166
375 -499
-16 188
265 458
-3 196
249 453
204 250
270 440
285 470
425 -126
53 -82
121 -53
10 9 26042 6
-145 483
453 -315
426 -6
-431 51
136 418
224 -502
316 -480
-265 252
205 -124
-214 -2
124 392
216 -110
411 -338
-93 481
-409 78
-207 26
278 -55
-421 57
17 309
9 10 32213 9
432 -394
306 12
5 274
65 216
384 -180
226 -320
341 -480
-56 -460
-501 -231
449 -393
383 -369
-342 -137
28 452
41 220
256 -364
312 -430
49 284
-11 -436
186 -287
9 10 3114 6
-59 -93
25 -232
392 -307
31 -27
-138 -47
103 101
-454 -326
-386 227
-454 -247
39 -34
-455 -324
-63 -90
-55 -100
18 -27
-62 -98
22 -245
101 115
-52 -88
21 -241
10 9 1962 5
-279 -317
414 389
478 429
-332 -10
84 -131
319 -481
-140 -296
456 470
417 291
246 -439
-101 -275
-133 -297
303 -484
-320 -6
307 -493
-138 -279
-144 -288
-138 -292
-338 -25
10 9 538 3
-479 -484
-74 103
-138 419
-138 359
-406 -249
-192 -186
258 -412
-68 57
6 243
-131 -452
-139 362
-65 103
-66 36
-483 -481
-68 108
227 -417
-132 401
254 -417
-485 -485
10 9 9197 5
-234 -254
487 360
129 132
123 -468
-125 -375
-302 -253
-153 -188
-10 -495
326 48
211 26
101 119
-132 -178
200 7
-210 -259
-144 -469
270 21
104 -461
189 28
150 -479
50 10 890 2
212 -442
475 -177
450 -193
285 -197
-206 212
333 390
511 64
406 487
418 -4
-81 408
303 286
-217 177
-503 341
-373 377
334 302
495 -130
267 102
288 -104
-143 -330
417 291
-54 -483
16 -483
-18 216
345 192
-446 -388
258 -487
74 15
75 -158
-66 378
308 502
-15 125
-473 263
152 -23
90 493
250 383
69 -329
245 119
159 24
6 80
398 -286
-157 215
-214 133
-92 -354
166 -20
410 45
-474 475
-472 193
-46 -267
228 -452
-467 -455
160 -26
67 -338
342 186
266 -483
81 493
247 111
-212 137
-12 116
459 -158
229 -494
50 9 5378 6
-393 -487
-68 343
343 326
328 35
-403 368
477 -407
-118 198
59 457
-137 64
-53 148
-415 -131
-54 324
253 132
-79 273
286 438
207 34
-175 -121
-284 -172
262 -511
229 90
-465 147
-480 85
427 426
379 -455
-162 -50
219 -92
107 -396
427 79
126 107
226 3
-423 -417
78 88
-459 -127
313 -76
-96 19
0 -32
-321 277
456 185
-238 -229
504 307
279 170
83 -315
-248 357
-212 348
-477 -471
126 -253
396 392
-344 140
92 338
-418 163
91 -307
-411 -435
77 -333
361 293
496 306
-193 -125
97 359
-169 -57
302 -60
54 9 6344 7
-358 495
119 391
252 52
-61 -203
-111 -267
175 -508
-132 -333
52 122
410 -195
-238 -245
-377 -509
-286 47
-316 76
-174 318
412 -26
124 428
397 -231
386 -333
-289 -412
132 203
415 -328
-93 212
206 471
-204 43
61 368
-282 -384
202 17
286 -403
-440 480
119 144
339 -297
-146 -43
504 231
219 -478
-226 -289
-125 -195
-80 390
45 481
-66 -34
122 224
-360 104
367 -15
-458 -89
277 -307
212 -411
-301 -81
-129 465
421 420
-387 313
462 -378
-103 -21
-51 -31
-104 167
-106 -187
441 -15
-137 456
203 -411
114 238
-215 32
-86 -21
-113 -198
420 -187
-329 69
54 10 8017 8
-435 420
203 -168
163 471
-472 60
-20 -26
-152 -129
415 156
453 271
511 290
-309 224
-80 121
-108 -334
448 -348
247 462
458 413
35 77
-418 -378
17 -336
388 54
-122 -497
-207 -174
-297 153
-487 466
-480 41
-88 156
294 -162
32 -501
352 237
-149 107
-116 427
-138 8
469 264
118 -74
-363 277
-119 -40
-492 491
308 361
237 -253
358 -226
-179 -144
364 434
198 18
-202 -29
510 60
147 -335
99 -230
-5 409
287 350
304 399
358 265
430 -96
245 198
37 -8
-427 477
289 -143
169 3
418 275
244 448
363 421
-131 -123
-415 -398
-23 381
-409 -397
-18 -42
54 9 2450 2
164 21
-449 427
116 205
-232 130
133 185
289 326
156 299
223 93
124 -177
-343 -88
144 -476
304 227
162 -165
336 99
-489 -214
113 163
343 307
-473 400
487 -470
-5 472
86 50
366 -52
58 268
-444 501
426 456
220 -282
-4 -123
-253 266
-260 -116
53 476
-42 -108
20 165
-236 60
482 351
-180 56
-491 -402
48 458
409 -156
118 185
457 -6
-342 280
104 23
-86 322
-490 -332
-177 476
-441 -74
-174 -299
-101 48
82 178
78 -482
-507 -410
-346 380
19 -356
-254 80
-182 -287
480 354
-99 325
-270 -110
223 107
-441 -76
160 287
-440 432
-171 483
54 10 4250 3
-392 496
83 510
159 -466
447 -143
-157 -386
228 421
-58 143
-97 -284
-129 392
-121 -327
-284 43
489 371
-494 -195
-245 431
402 10
148 -404
266 367
-298 -167
445 -277
137 -259
-476 173
351 -40
-447 399
360 -307
-129 99
121 357
-422 281
187 34
184 -100
-123 163
281 -491
-443 -167
-345 -81
-415 429
85 -493
-171 -394
116 241
-140 -209
-171 -69
-198 -92
492 -87
-311 392
417 -22
-276 195
1 -467
161 -509
-77 51
-389 270
-183 -72
-360 323
-476 -390
207 15
441 477
-87 -136
193 38
-422 276
-440 -163
409 -14
140 -409
-185 -398
115 357
-165 -60
446 472
-78 36
53 9 3298 4
-132 126
-270 52
398 -160
-429 383
-1 250
291 -479
511 -36
41 -456
252 -483
-424 -90
-321 -226
-50 -268
313 207
-134 141
-286 -281
459 358
-61 359
347 293
45 326
429 -48
220 -401
-263 -134
356 -311
213 -415
283 -134
-443 -413
9 -275
-197 131
-420 -103
267 254
-340 -305
153 171
-294 172
428 274
457 -386
-37 458
221 -308
-406 -399
-253 -482
-249 46
-464 -498
13 -55
-214 -50
-434 41
-188 72
483 441
301 128
44 -394
-335 333
-420 -4
31 -480
-471 -133
-264 -451
-474 -123
-251 -453
434 -39
-456 -507
261 -479
-473 -503
-468 -141
-446 -407
-191 80
51 9 1202 2
-190 270
448 398
367 147
-311 -393
159 -422
62 -125
40 -130
476 -135
-232 212
-392 -396
336 -102
-319 -355
-360 354
-83 396
444 237
151 -496
-426 75
-298 509
-118 479
206 260
-410 291
454 415
-437 -80
165 -372
310 378
-388 -124
-87 -393
-286 268
-191 -91
-117 398
4 -33
-62 93
-152 -380
-149 -110
-115 118
-364 87
183 343
315 -149
-335 88
392 -31
266 -204
29 -249
474 134
-267 256
301 201
-301 412
-383 465
367 141
-435 179
441 -336
-46 -185
-108 398
-278 258
-293 414
-81 64
323 -172
471 -134
400 -31
-60 104
-426 -79
45 10 8369 9
-480 284
-137 124
-240 79
260 -56
2 362
201 24
365 -437
-401 214
337 -161
422 169
325 -227
-98 -170
76 -11
-305 129
369 288
471 390
-496 158
354 293
-1 110
180 3
139 7
-286 508
-280 -30
280 348
-192 -135
356 -28
478 354
-96 493
-118 403
440 -177
-50 -243
334 -320
-287 -494
-328 -157
-51 74
-388 17
24 349
-284 264
-489 -126
-55 232
118 6
-101 430
509 57
-376 60
-203 91
246 -295
-274 291
-79 213
-78 416
149 -5
496 62
337 -184
272 -32
-314 149
-304 -492
800 80 22733205697 21
3730837 1279448
3648214 -2562426
-989418 -1402808
-4042397 -2856050
-961372 491090
-3063810 551558
1611013 -2627159
-2039903 209975
-1100083 -3483462
-375548 -343993
-3850340 3237523
3664943 -3899230
3648993 -2219480
3891840 -2390202
-2347441 -3479831
-1104766 1889355
-2108132 -2674560
-730089 2698630
-2202490 -456722
2139965 3400355
-3703168 3118936
2231213 -3126395
2272400 1174626
962049 -3043020
115872 -695948
-3404449 -2151931
-52501 2937248
-2068004 4088437
-3746655 -370133
-112064 424360
-4056419 2105520
129143 751186
-1883268 578481
2885151 -2081562
803096 -1246319
-2986454 -3278787
2579154 506490
-3394653 1467374
2416665 3664985
2230180 -3877263
890989 -3272962
-4190536 741697
1572253 1102634
-1161421 -3161654
-2190 2667329
-3188588 -2407902
2105021 2667250
341378 2770934
-3515003 1753664
925775 -64994
-95915 3988363
2918038 2284753
-1787839 625294
3735372 4000711
-3887557 -3249985
-1437735 -2084856
1323222 2298298
1188028 2302847
-1942451 -3527353
1494016 -3160657
2433760 -171453
-3801393 -804647
-2187323 -605401
2482735 2888013
-3987112 -3290723
-367258 1835683
-1063770 1448006
-1627956 162282
-1924641 3172402
-679807 2777591
-3571508 18381
3888838 -1373314
1936665 200493
-209492 1905982
-724658 627
729961 -897847
298605 -1103726
3047356 -843523
-2500232 -2962389
3746606 3123518
397584 332807
-1342201 3542334
-2191824 -3813132
-2898042 -3058540
-3102776 -406601
1439236 -2547088
-97207 806853
-3805885 1565083
-3413912 -2826275
-999822 3576934
-1822931 3589161
4148071 -574640
215067 1638078
-1885284 -2296350
-3564879 590755
396560 3847371
1570670 1487478
3879998 808713
-341514 2891382
-97112 -699175
-2214721 -2359708
-1567005 -3037687
-3587322 -405385
2731684 3160308
-2656122 -2323879
-147217 -2443585
1673200 -3740434
-3728838 464441
-3503443 -859935
-1254608 -2229381
-1852588 3277010
329735 -1310235
-718034 -2831614
-1360100 2213151
2663987 -99853
182394 2707802
-3270198 3176896
-597619 1615774
-2806060 4128271
-2660558 3306822
3809848 2223838
58448 3791220
3159329 3768308
2462956 -1237733
2590116 -4143525
3781428 -171571
-2239006 2240946
-2160459 -190086
385663 -3491773
3858917 -1724905
2742366 -4159
3298280 -2587756
-429693 -589058
1549804 -1876300
-2945697 2910458
345391 -2268683
3182676 -3572263
-219041 -2344545
459840 1305614
-1180523 -3905165
927524 -3051837
-1724311 160949
-2323174 1897377
-1571259 2897906
2391445 183665
3341008 -382155
2429169 -3280582
-937252 3829941
-237930 -296716
23332 -2199984
459828 1078259
-2966374 -3547784
3379529 3429125
1398934 -1251370
-1143535 193623
-1814668 -289830
-260056 -1305530
2020108 788970
3918341 756457
1091930 -916598
1944477 235468
-2687185 -1904669
1786949 -600166
3284618 2131168
2140631 -2041579
1888710 3873120
-3029303 31638
-3139104 1522869
-2907438 -2245806
-3326896 -1178214
-736371 -406946
1363394 2759779
122835 -3120531
-1666274 -2211327
3309505 4026451
-381843 3094563
2125526 2695776
-499793 -3425870
-1111932 -3183185
2983808 1317777
-2794577 2845348
2200016 -1249756
2041395 3539405
2191951 -3118890
4004436 3676088
1428968 -261841
556304 -3695217
1249385 -3518165
2889729 85248
1257159 3699498
-3512673 2210799
2676354 3978138
3235312 2139758
-1522375 2505616
-3421166 2241817
-2541698 -2125840
908924 -3202706
-567409 -1458270
-1010848 3265100
-2710529 -3694586
-163854 -3465155
-3811046 2088589
372921 -1669480
161494 -4007089
1581133 2917939
3051388 -2183562
-2044694 3678798
-63509 -1126488
1168480 3889921
739530 788781
2773323 3990256
-2853997 3811801
566470 -4023253
-2450390 -571051
-235476 152641
69861 -1515435
1730110 -3470735
-2010116 -3075005
-2501703 3118873
1447148 111945
2316631 -3603902
-176297 -3467125
-2729007 2599212
3776594 -1027187
3849000 -615310
3834080 2878332
-3178348 3948811
2707708 -3828936
-1322403 -2828923
-3557502 -2529166
-649781 1775570
-1307333 -3644145
4052877 -1073941
3753274 -2516656
-213083 185618
-2762675 -1447235
-436896 1992283
2303863 -521215
-1511121 -2197017
1844800 3458419
1365917 -1114290
3247976 988754
221155 -3851237
2650652 1306226
2722723 1888604
3082719 2734044
-2665979 1191499
1846990 -1424527
-214084 -3289772
1583205 -891783
-2667361 -3342875
-1517039 471180
-3870181 725490
2258371 1530800
2720276 -2660521
-3623811 -2322696
-1706557 288903
-3605715 -1990694
479124 -1813181
-3840223 -85191
-3654495 -324268
-155729 1967306
-2724158 -3148108
3874154 1245400
1621133 4068044
-466720 631658
-2659835 -3181706
-1156892 -4159476
4085822 -3256673
3244100 2155036
2304355 285838
350086 2026201
1104339 -1978396
-2518480 -2717146
3376423 4167413
-1925612 -1047353
3477448 3054067
-3884380 3520046
3401523 2448924
-3285278 4007747
-3248581 3943882
-3982807 -185878
2700877 2921102
159773 -1725759
-289383 -2145460
2043977 -3714451
4081142 -1120767
-2971108 2992301
1884910 -2068217
-585533 1556107
3971719 2094123
-1302086 -2994180
1842764 -2265157
1873981 3960401
491156 76992
3819693 -2941692
820006 505082
-1667506 2637482
896877 -4045626
3236500 2806066
-2877709 -3484101
-3546718 623760
2021220 1106094
1613351 -2040396
-3196209 3736713
3123881 -395962
2330067 1167352
917374 -917432
-584703 2280712
-2525145 1909010
1426676 2221079
449835 2544455
-1096391 -2914780
-2952763 3442879
-3526086 3780148
2173667 -3330320
1655976 -1452260
-188543 -3728924
543310 3817680
-1911333 -3746073
1054555 3593635
2880551 -2650546
-3081995 2866833
1113571 3057723
3882525 3443451
-2417997 -3883736
-813866 2281160
-2022928 -3878254
346946 698575
-1431710 -2985967
314593 3323497
-2936520 -2283077
3562114 2703755
2005206 -2488384
2989727 -1746640
2008504 -2140922
-839148 -927856
-3049983 -1822392
3346055 2033296
-2802065 -2806188
1766565 3957871
-94026 1223484
-2561770 -720340
-273718 -1680554
-3916533 3495971
3066222 2438265
1941326 210021
2214961 -687677
-1374134 3069268
1756196 -1720884
-293063 2975831
-3694572 -1412486
4026765 -1280732
-1324956 763965
3987223 2194543
-1568628 -1687206
1243943 -2596096
3452258 2381841
23804 -2418740
1938042 -3415298
-759917 -675508
2525792 2952784
-905526 2481826
-3345858 -852820
3912212 -3524217
-3410454 2177315
1873655 2451054
1538501 1935053
414381 -2435755
511068 -1184481
-3930692 2667010
154436 -3704492
-3340308 3132372
2060121 2275249
-2216331 3897644
-3609222 2009383
2559989 -3381734
-741624 2523036
-2267379 2084113
319938 -3599410
1273583 2277543
-192246 -1784166
-65950 -1303432
-322172 2959926
-1898954 -2033282
895804 3140489
622349 -2899210
3549274 -2555317
-3732564 -3018477
-2104376 396452
390170 -2936615
-1457892 -1079005
-111482 2730023
-55343 3883376
2931052 1471251
2392641 -2135501
-396050 -1689028
-2146807 -2848085
2220324 821971
3207267 -2014846
-559937 -2625182
-470099 4068220
-1152902 -3142207
1171547 2760432
1866968 -2735351
-85437 2600512
-3815289 4110705
-624605 -743670
-3932312 651424
2518760 3652574
2724672 427815
-2182021 -2277011
-628456 -665689
-3473387 -3230642
-1308811 -3317809
-2908196 1774096
1550193 426411
2860434 3320790
1528657 808724
967549 2798425
-1068216 3290852
-2387198 818379
-2592580 -143
2933327 -1021419
1777731 -2133016
3084615 -2006195
302401 -387483
-163379 -1470046
-435161 3132310
2950140 1379626
-50190 -420690
2850649 1586799
2895268 -184303
-74389 3593496
-765196 487790
1262389 979484
2285643 -4029955
66481 2992473
-2241487 -4121303
-1782275 3297362
3629898 718382
714128 2718744
-2317054 1144869
-339835 3638535
-1008465 -2591229
-2983949 2344571
-3563296 -544879
758497 317028
1245235 -2436863
968955 341947
1542899 2167531
-827111 -3485049
4040057 -1117043
-3341624 -1608267
2225673 -2326331
-3937631 3045728
1317077 3055705
1690527 3671370
-3376282 1464811
1063332 -3225997
1290377 314139
3093021 2176257
-233742 -118239
-83047 2439499
2130443 -766880
-1469838 1986552
-1966454 -1955135
1657678 775528
-689322 1330744
-1865560 1157923
-557771 -2973111
2572490 1735016
228222 -1313504
-1294860 -1505673
3591980 1760568
1792807 3728794
2925688 4176818
-3523313 -3959139
-4164675 -1870373
-3753262 -898563
3418003 -2063251
2839151 -885668
-3433233 -1442772
262461 3777219
3969700 -2469481
-3124679 1938461
343900 33618
3942786 -712292
3707285 2204495
-3375164 -1780475
-2586125 -2315602
483093 -3872430
-4108624 -3419779
-4159081 1484669
-2029974 1292959
3652303 1071005
-1322431 -1458181
-51506 4031470
1605924 3828273
-2720447 -941406
-1586116 -161178
2779877 -113043
-2007345 3791310
2616216 1486819
2357546 24451
1536273 1216272
3559303 3022067
-2621008 3241930
-4093821 489234
-2608470 -501749
-3344580 -1504899
-3721976 2849106
131689 311924
-1324533 2009587
-2261935 2337633
1673864 -884280
-3911182 -475117
-1635361 -3178009
-1480965 -1778679
-3346228 -1846450
4082860 566116
-2956256 3705431
2887221 -3268009
-3648011 -2397290
1533655 -3658280
9438 1120767
556014 -2636196
-2681578 3125253
-1611414 -2944474
2478282 -2649715
-3222611 -617834
3898412 2679213
-3847439 1228296
-2024087 921385
2073722 3253837
504764 499195
2370978 3780353
-881879 -526518
-948083 1519687
-3168366 175015
2901946 1016712
1802026 -2274966
-2709785 -4007631
-40413 -278535
-3998073 -2034462
3388358 2412926
-2577735 -493613
1924254 -2846423
-1714984 2296506
-4189296 2402339
2490168 -1207861
55887 1148198
3740589 -2756553
-2090492 -620873
-1872564 4050774
3273778 2680870
-1826914 -1993234
1247048 790691
-2697486 2355772
2319989 -1857508
1627174 -3555766
2543146 -2155042
3944643 1892630
-171062 -3457231
3818946 -425001
2284984 -3042641
-1940 396415
-2502620 -2455478
-3727694 840041
3690260 -1697778
3952340 3123480
-2626255 -2570797
1021676 83778
2717691 -3247153
-2102842 -2095906
2833734 3135131
1640102 361857
-1138481 -32808
-822644 4170258
-1288978 1397935
4013823 -2199060
-3011466 -769775
-777667 -3861510
-4116975 -3798821
1012590 659463
849847 4050473
255985 1063447
-690422 -2519458
-1852033 -1882278
1829017 -3404683
-1529300 -3786835
-2197685 -795433
-2722660 992620
2081874 -1058157
-3258043 -3886696
-2669969 -3478504
-1104614 -1112045
-1424837 2521435
-2917412 -2089487
2080248 -3481634
-564612 -1226770
3205098 673537
2454041 -3306706
-3872207 506300
-133619 -2630912
-1068904 3552242
-2262610 2104812
-2853232 -3868923
1345171 2170164
2649620 30954
1980885 -3001749
2639409 3159700
-288341 -1444428
-1845430 3540479
-3736320 -580812
629120 -2208148
-3619820 2975489
3134398 -1266153
3276839 1470637
3058712 -1225560
513133 -3821505
1930857 3900006
3914374 -3528775
-239797 2346157
-1458487 -2445270
-3903787 -3217621
-2365318 2246567
1602556 438802
-3803164 3553462
3552399 -2442251
2495283 1729132
1218745 -2971147
-825760 -1571817
4153741 -2121182
-3611026 -2950962
-3519941 -1458619
-844068 127385
1693599 3768746
3470666 2024089
1873746 4079964
410158 -112671
-3041016 -1363595
-2535782 2916924
2712918 1318303
1546445 3779899
-3230897 1626943
-3189894 31934
683660 -1797282
4141599 -348726
-1409054 -1197513
-1242619 2915706
3069576 81023
-1526183 -974153
649974 -2810871
3422180 3119541
-806651 -3304571
-904119 -1295311
936139 1779712
-72855 -3114269
472595 2272838
2850469 -2275565
313043 1905757
1241397 -472242
1121928 2412026
-1035873 -2990285
-628217 2821616
-108274 3082167
-3979097 4023667
3963489 -211009
1291483 -3441069
-2761119 -1103809
3406736 2528693
-1164375 -2157448
3272570 -2187479
-3078248 -1316412
-910815 -715407
-3029137 2225200
4026377 -4078611
-1673228 462479
2257902 2298307
-675236 -4092189
-2827827 2411878
-403735 4072933
-2686869 -2014974
-1704511 -3314028
-3633873 -171967
-1001432 -1609439
-2419894 682898
3129895 -3199810
2673771 -2127308
-895155 1375718
1417843 -2520477
-1222199 1090189
-874707 -496133
-1196496 -3984660
-3708214 -2112648
1789626 554890
2645741 2465132
3142538 2438353
-2478748 -1207838
2601942 -1964137
971564 -2363364
593466 -2278392
-282232 -88158
455770 1009420
-2069249 1937173
633458 218429
-885236 -3174427
1770071 2708499
-3877767 2381105
-1835957 1673567
-621181 4005557
624074 -564745
-2860720 435796
1669121 -1816116
1998733 816255
-2707105 -468608
2280948 554160
1048516 -1679509
-3730225 -2646664
-1110480 509098
-608855 -2444196
1625411 -3616274
1405205 -1451440
734124 2294841
-898023 842168
2235377 -1098607
-3086452 1697143
-684534 -3042581
3217555 -3886664
714848 1015191
945144 662357
-933760 4104204
-1499040 141406
-2006846 1691588
-522984 3767682
-2215347 3152977
-2388533 3103264
2745498 -3594402
3669966 -2377312
3534867 -1821053
-5339 -1719754
-3981015 -1074222
2514606 -2605428
-353050 2105526
3542168 -3852706
-40003 -2705407
-2608173 547486
-3586711 3223510
287704 1401084
1809984 1271134
2075558 -331535
402062 -3940469
60797 908662
-3717606 -2522621
187784 -1855315
-1083924 -745729
-960268 2557349
2941918 -3082347
-227915 768302
-3325781 -3620410
2734309 2001003
1907990 -1151609
493971 2852172
-1166224 3869682
516765 -74284
407549 -778947
-2747432 615156
1275560 -1326339
113637 1553302
-1087742 666040
-3580565 -2987814
-409810 561402
1190900 2751741
3398558 1946504
-2597939 777354
1685726 -442098
668685 -3572046
-3321800 -1102748
-2101847 -1777770
-3573339 1452156
-3053480 -3161532
-1616665 -278374
2414451 4063083
-3885740 176631
3591244 -1487257
-1229683 -3016866
-1429841 -3955837
-2528540 -1780795
-3321936 2488825
2056715 197086
4005196 1078926
3303321 1153243
2561476 -2484877
2410732 -2349427
-2211202 -799136
-1556640 1536823
2875104 -614207
-1979287 3445484
-2317590 3329872
-1159290 -3874718
-3606707 -2983251
4042647 -2197686
-3858438 3271380
-624642 4030932
-1284703 2993666
-834095 2364733
-2578284 807484
-2697159 -2002141
-1229855 -3983856
-1146456 -10713
480945 -3823975
-3605491 1565927
-2957137 -2308348
1566271 1227996
477474 1035227
-3964976 -3277723
1853303 -2760644
1470713 -2544168
-90681 1195413
-3774508 -810640
3891635 -1393152
-1547645 1527417
2066781 230635
3023938 -1224399
-469880 539751
-3740564 -2625082
-857085 145349
-646771 2826589
4044351 -2187408
3969796 2109921
-3297494 -1127511
-1108904 -1144030
3783768 -2501327
1270490 200982
-3676080 -2386655
-230033 -147468
-1652556 458465
1029927 662372
3032726 -2175871
-601300 2304145
1577606 -2039753
-3813623 3565021
-2894872 1772019
3084716 60018
-1414831 2522320
-246915 132214
-2719398 2613961
-2223471 3919467
3143366 -404300
-1584380 -156367
322354 2013397
2708648 1340893
2489636 2874918
-2561006 -725712
2007144 3284655
-4114292 -3445648
451828 1277301
-652733 -4083959
2507043 2937580
-351645 4096906
-2829260 -2809396
976964 -3013707
-2721807 -918719
-601880 1620069
-742488 2501723
2224649 -1112544
-303373 3648295
-904677 2457802
-1679058 2929329
1792348 543028
2849464 3133654
1964513 804473
2022951 3552666
-2551080 -483995
-1092291 -2937551
-1758475 613144
-3713966 -370766
-3789542 -2588443
3804037 -179092
