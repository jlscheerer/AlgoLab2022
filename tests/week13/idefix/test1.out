3 113480
3 5776
2 47780
4 49040
2 104168
3 128852
4 12456
4 7848
2 2152
2 36788
1 3560
2 21512
1 25376
2 32068
1 9800
1 17000
2 13192
1 4808
1 33476
2 90932822788
